src/main/java/xbot/common/subsystems/drive/PurePursuitCommand.java

```diff
@@ -124,6 +125,10 @@ public void setPIDsToDefault() {
         this.headingModule = defaultHeadingModule;
         this.positionalPid = defaultPositionalPid;
     }
+
+    public void setDotProductDrivingEnabled(boolean enabled) {
```

Comment:
I'm wondering if this should be named for the behavior it creates rather than the mechanism by which it achieves that behavior? Like what if we find a better way of doing this than dot-product or maybe some variation thereof?